spec/services/email_settings_exception_handler_spec.rb

```diff
@@ -38,7 +38,7 @@
     end
 
     it "formats a Net::SMTPAuthenticationError with application-specific password Gmail error" do
-      exception = Net::SMTPAuthenticationError.new("Application-specific password required")
+      exception = Net::SMTPAuthenticationError.new(stub(message: "Application-specific password required"))
```

Comment:
I'm curious what is the context for this change, is there a way we can do this without stubbing?

Reply:
It seems the code from `net-smtp` changed a bit: https://github.com/ruby/net-smtp/blob/bf31db88659bc8462de5921ae7a7ccefd87d98e8/lib/net/smtp.rb#L37

So now the first argument can’t be a string anymore but needs to be a class that responds to `#message`. We could probably write it like this `Net::SMTPAuthenticationError.new(nil, message: "…")` but it could be a bit confusing too :sweat_smile:

Reply:
I think it is better to go with `Net::SMTPAuthenticationError.new(nil, message: "…")` than providing a stub to the error.

Reply:
As you wish, I don’t see the matter with using a stub in this case though.

Reply:
Is there an advantage that the stub provides over passing the right parameter to the error?

Reply:
I don’t think there’s a clear advantage for either of the options here, just providing what the exception class now expects :man_shrugging: 
I’ll make the change anyway, I don’t have a strong enough opinion on this :grin: